lsp-ui-doc.el

```diff
@@ -304,7 +304,7 @@ Because some variables are buffer local.")
          string))
       ;; For other programming languages
       (language (lsp--render-string (lsp-ui-doc--inline-formatted-string string) language))
-      ;; For default markdown content
+      ;; For default element content
```

Comment:
I thought the default would be `markdown` content. I guess this would be more precise.